src/unix/getaddrinfo.c

```diff
@@ -200,3 +200,22 @@ void uv_freeaddrinfo(struct addrinfo* ai) {
   if (ai)
     freeaddrinfo(ai);
 }
+
+
+int uv_if_indextoname(unsigned int ifindex, char* ifname, size_t *sizep) {
+  char ifname_buf[UV_IF_NAMESIZE];
+  if (NULL == if_indextoname(ifindex, ifname_buf)) {
+    return -errno;
+  }
+  {
+    const int required_len = strlen(ifname_buf) + 1;
+    const int ifname_len = *sizep;
+
+    *sizep = required_len;
+    if (ifname_len < required_len) {
+      return UV_ENAMETOOLONG;
+    }
+    memcpy(ifname, ifname_buf, required_len);
```

Comment:
Why?  `strcpy` is unsafe.  I could use `strncpy`, but since we already know the length of the name, why to use `strncpy` which is less efficient that `memcpy`?